packages/asc-ui/src/components/Checkbox/Checkbox.tsx

```diff
@@ -55,4 +58,6 @@ const Checkbox: React.FC<Props & React.HTMLAttributes<HTMLInputElement>> = ({
   )
 }
 
-export default Checkbox
+const CheckboxAny: any = Checkbox
```

Comment:
We do not have to use any here, we can directly wrap the component definition in a call to `forwardRef` in order to get all the type information we need:

```ts
const Checkbox = React.forwardRef<HTMLInputElement, Props & React.HTMLAttributes<HTMLInputElement>>(props => {

})

export default Checkbox
```

`forwardRef` takes two [generic types](https://www.typescriptlang.org/docs/handbook/generics.html), the first being the type of the element that is forwarded and the second the type of the props of the component. For more information check out this [blog post](https://medium.com/@martin_hotell/react-refs-with-typescript-a32d56c4d315).

Reply:
gedaan